cmd/kube-apiserver/app/server.go

```diff
@@ -68,6 +68,7 @@ type APIServer struct {
 	BindAddress                net.IP
 	AdvertiseAddress           net.IP
 	SecurePort                 int
+	ReadWritePort              int
```

Comment:
I'd rather infer you want an insecure cluster by `--secure-port=0 --insecure-port=8080`